src/IECoreImage/ImageReader.cpp

```diff
@@ -429,6 +438,29 @@ class ImageReader::Implementation
 			}
 		}
 
+		// Returns the miplevel to be used for file access and makes sure that
+		// the file can provide the respective data.
+		int miplevel() const
+		{
+			int result = 0;
+
+			ConstIntParameterPtr p = m_reader->miplevelParameter();
+			const IntData *d = static_cast<const IntData *>( p->getValue() );
```

Comment:
No need to muck about with data here - `int result = m_reader->miplevelParameter()->getNumericValue()` should do the trick in one line.

Reply:
Fixed in dffb0de